packages/openid4vc/src/openid4vc-holder/OpenId4VciHolderService.ts

```diff
@@ -1070,7 +1070,9 @@ export class OpenId4VciHolderService {
     this.logger.debug('Credential response', credentialResponse)
 
     const credentials =
-      credentialResponse.credentials ??
+      credentialResponse.credentials?.map((c) =>
+        typeof c === 'object' && Object.keys(c).length === 1 && 'credential' in c ? c.credential : c
```

Comment:
I think the reason for making it an object is so that extension can be added for each credential. Checking for a single key break that.

How about checking that every item in the list is an object and has the `credential` key, and then we use `.credential`, otherwise just `c`

Reply:
Done

Reply:
Done!